### DISCUSSION TITLE
News Section Improvements

### DISCUSSION THREAD

**Discussion by turkeysanwich** [trakt/showly#777] [Ideas / Feature Requests]
I think there are some simple things that could make the news section a lot more useful:

1. Prioritising news about shows/movies in your collection (this could be in the form of a sort option that only includes entries with your shows/movies titles in it somewhere, or just highlighting/pushing to the top articles that mention stuff in your collection)
2. Chronological sorting (show posts in order of newest to oldest)
3. Some more filtering options besides just Shows and Movies (e.g. trailers, interviews, clips)

**Comment by michaldrabik** [comment]
Surely am planning on expanding News with additional stuff (like also bookmarks for example). Will keep your suggestions in mind when I'm at it.
